### PR TITLE
Added Cisco Telepresence Peripheral to dhcp_vendor_class

### PR DESCRIPTION
**Pull request by modoyle-r7**
## Description
Added Cisco Telepresence Peripheral to dhcp_vendor_class

## Motivation and Context
Improved coverage

## How Has This Been Tested?
 ./bin/recog_verify xml/dhcp_vendor_class.xml
rake tests

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have updated the documentation accordingly (or changes are not required).
- [x] I have added tests to cover my changes (or new tests are not required).
- [x] All new and existing tests passed.

### CODE REVIEW

File: xml/dhcp_vendor_class.xml [hunk 1]
```diff
@@ -518,6 +518,15 @@
     <param pos="0" name="os.vendor" value="Cisco"/>
   </fingerprint>
 
+  <fingerprint pattern="^Cisco Telepresence Peripheral$">
```
Comment (mkienow-r7) [inline xml/dhcp_vendor_class.xml:521]: Is this a particular device or a family of devices?

File: xml/dhcp_vendor_class.xml [hunk 2]
```diff
@@ -518,6 +518,15 @@
...
+    <description>Cisco Telepresence Peripheral</description>
```
Comment (mkienow-r7) [inline xml/dhcp_vendor_class.xml:522]: Minor: update the casing in the description. I understand the DHCP vendor class value has the above casing, however, from a scan of this Google search (`Cisco Telepresenc site:cisco.com`) it appears that Cisco uses "Cisco TelePresence".

File: xml/dhcp_vendor_class.xml (hunk 1 above)
Comment (modoyle-r7) [reply xml/dhcp_vendor_class.xml:521]: Agree, family is more suitable, changed.

File: xml/dhcp_vendor_class.xml (hunk 2 above)
Comment (modoyle-r7) [reply xml/dhcp_vendor_class.xml:522]: Done.

**Review by mkienow-r7 — APPROVED** [review]
Thank you for the contribution @modoyle-r7!

**Pull request merged**